dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/UsersServiceImpl.java

```diff
@@ -151,11 +158,16 @@ public Map<String, Object> createUser(User loginUser,
         //check all user params
         String msg = this.checkUserParams(userName, userPassword, email, phone);
 
+        if(resourcePermissionCheckService.functionDisabled()){
+            putMsg(result, Status.FUNCTION_DISABLED, msg);
```

Comment:
After the interface of the User module is verified using the functionDisabled method, no other permission verification is required.